### DISCUSSION TITLE
✅ Show the Name field as the Item title in the Slides list of the Testimonial Carousel

### DISCUSSION THREAD

**Discussion by maxms** [elementor/elementor#13920] [Feature Request]
Hello,

This is a small request to show the name on a testimonial as the item title rather than just Item #1, Item #2, etc. This will make it so much easier for users to organize and edit in the future. Otherwise, you have to open each item and try to remember which is which.

For the Media Carousel, same problem with identifying each item. A custom title field would be very helpful.

Thank you.

**Comment by nicholaszein** [comment]
Hi, you can do this by using the new Nested Carousel widget. Each Item has its own Title, then all you need to do is to create your testimonials and design them the way you want.

We currently have no plans to improve the Testimonial Carousel widget, as the Nested Carousel has better functionality and is future proof.

You can learn more about the new Nested Carousel in this article: https://elementor.com/help/carousel-widget/
